_processSteps.py

```diff
@@ -279,45 +280,25 @@ def submit_job(argv, inps):
        if inps.wall_time:
           f.write('#BSUB -W ' + inps.wall_time + '\n')
 
-       #f.write('#BSUB -W 2:00\n')
-       #f.write('#BSUB -R rusage[mem=3700]\n')
-       #f.write('#BSUB -R span[hosts=1]\n')
        f.write('cd '+inps.work_dir+'\n')
        f.write(command_line)
        f.close()
 
        if inps.bsub_flag:
           job_cmd = 'bsub -P insarlab < process.job'
-          print('bsub job submission')
+          logger.log(loglevel.INFO, 'bsub job submission')
           os.system(job_cmd)
           sys.exit(0)
 
-##########################################################################
-
-
-def step_dir(inps, argv):
-    inps.project_name = putils.get_project_name(
-        custom_template_file=inps.custom_template_file)
-
-    inps.work_dir = putils.get_work_directory(inps.work_dir, inps.project_name)
-
-    # Change directory to work directory
-    if not os.path.isdir(inps.work_dir):
-        os.makedirs(inps.work_dir)
-    os.chdir(inps.work_dir)
-    logger.debug("Go to work directory: " + inps.work_dir)
-
-    command_line = os.path.basename(
-        argv[0]) + ' ' + ' '.join(argv[1:len(argv)])
-    messageRsmas.log('##### NEW RUN #####')
-    messageRsmas.log(command_line)
-
-    return inps
+    return None
 
 ###############################################################################
 
 
```

Comment:
create_or_update_template function is added at the beginning of each script to read and update template files